.github/workflows/pull_request.yml

```diff
@@ -57,20 +57,31 @@ jobs:
   build:
     name: Build 🛠️
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - 'linux/amd64'
+          - 'linux/arm64'
+          - 'linux/arm'
+          - 'linux/arm/v6'
 
     steps:
       - name: Checkout ⬇️
         uses: actions/checkout@v2.4.0
 
-      - name: Setup node environment ⚙️
-        uses: actions/setup-node@v2.5.0
-        with:
-          node-version: 16
-          cache: 'npm'
-          check-latest: true
+      - name: Configure QEMU ⚙️
+        uses: docker/setup-qemu-action@v1.2.0
 
-      - name: Install dependencies 📦
-        run: npm ci --no-audit
+      - name: Configure Docker Buildx ⚙️
+        uses: docker/setup-buildx-action@v1.5.1
 
-      - name: Build ⌛
-        run: NUXT_ENV_COMMIT=$GITHUB_SHA npm run build
+      - name: Build Docker images 🛠️
+        uses: docker/build-push-action@v2.6.1
+        with:
+          context: .
+          platforms: ${{ matrix.platform }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+          tags: |
+            jellyfin/jellyfin-vue:pr-build
```

Comment:
What does that do if nothing is pushed? Is it for the cache or just because the action requires something? If the second answer, maybe add a comment to say so :)

Reply:
Just builds the image to see if build succeeds. I removed the client build step altogether because we can assume that, if Docker images build, client will also be built :).

Reply:
Np about that, I was just wondering what that line did in that case :)